extension/vsc-ts/howdoi/README.md

```diff
@@ -0,0 +1,65 @@
+# howdoi README
```

Comment:
This readme looks not quite finished. If we decide to go with the `vsc-ts` version then let's make sure the README is up to date